packer/ansible/roles/setuptools/tasks/main.yml

```diff
@@ -4,3 +4,6 @@
 
 - pip: name=setuptools extra_args='--upgrade'
   sudo: yes
+
+- pip: name=virtualenv extra_args='--upgrade'
+  sudo: yes
```

Comment:
Why upgrade virtualenv in this PR?

Reply:
in ova venv version is too low.
And with such venv,  pip install  in ```vnc_record.sh``` will throw error.

Reply:
I think it's better to modify it in PR for build_ova.sh

Reply:
but without the ova with upgraded virtualenv, current ova post test won't work.

one pr or two pr is both ok for me ~  but I think this is not a urgent topic, merge this PR and let feature branch work is the urgent affairs